content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -281,6 +282,7 @@ For a reference to old feature gates that are removed, please refer to
 | `JobTrackingWithFinalizers` | `false` | Beta | 1.23 | 1.24 |
 | `JobTrackingWithFinalizers` | `true` | Beta | 1.25 | 1.25 |
 | `JobTrackingWithFinalizers` | `true` | GA | 1.26 | - |
+| `KMSv1` | `true` | Deprecated | 1.28 | - |
```

Comment:
Are we saying that setting the feature gate to false is something you don't recommend doing?

Reply:
Do we need to add KMSv1 to the [Deprecated API Migration Guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/)

Reply:
I do not understand this question.

Reply:
KMSv1 is not a ReST-like API and so we don't need to help people migrate away via the API migration guide. However, a separate migration guide would be very useful to write and publish.